src/BlazorStrap/Carousel.razor

```diff
@@ -1,6 +1,6 @@
 ﻿@inherits BootstrapComponentBase
 @using System.Timers
-@using  BlazorStrap.util
```

Comment:
This caused a compile error!?! Weird, mine compiled fine when I released 1.0.0-preview6-17